### DISCUSSION TITLE
Google Services

### DISCUSSION THREAD

**Discussion by ihossen016** [dailydotdev/daily#436] [Feature Requests]
You guys should add options to use Google's services like Gmail, Meet, Docs, etc. Installing your extension hides these service options from the new tab. It'll be great if you add them so that users use them easily.

**Comment by AnishDe12020** [comment]
You can add the most visited sites here already ([Docs](https://docs.daily.dev/docs/customize-your-feed/preferences#show-most-visited-sites)) - 

![image](https://user-images.githubusercontent.com/63192115/150459819-91e7e939-fc91-4428-beee-68a4095f0d84.png)

However, an option to add custom links sounds like a good feature request.

**Comment by sshanzel** [comment]
Hey to both of you! Hope you all are doing alright!

Thank you so much for the awesome suggestion but worry not! We are almost in the finish line with the feature to allow users to have custom links of their own. Stay tuned! 🚀
